pyproject.toml

```diff
@@ -31,6 +31,7 @@ boto3 = ">=1.28.61"
 # See https://stackoverflow.com/a/76647180
 torch = "2.0.0"
 pdfminer-six = "20221105"
+python-pptx = "^0.6.22"
```

Comment:
Do you still need this if you are using unstructured?

Reply:
Yes! The tests were failing without it.